wolfcrypt/src/pkcs7.c

```diff
@@ -8498,8 +8508,21 @@ static int wc_PKCS7_DecryptKtri(PKCS7* pkcs7, byte* in, word32 inSz,
             }
             #endif
             if (ret == 0) {
-                keySz = wc_RsaPrivateDecryptInline(encryptedKey, encryptedKeySz,
-                                                   &outKey, privKey);
+                #ifdef WOLFSSL_ASYNC_CRYPT
+                keySz = 0; /* set initial "ret" value to 0 */
```

Comment:
Check indentation here. Please add comment here that this makes the call blocking. Same for pkcs7 change above at 6499.